src/ruvsarpur.py

```diff
@@ -258,39 +258,52 @@ def parseArguments():
   parser.add_argument("--desc", help="Displays show description text when available", action="store_true")
   
   parser.add_argument("-d", "--debug", help="Prints out extra debugging information while script is running", action="store_true")
+
+  parser.add_argument("-p","--portable", help="Saves the tv schedule and the download log in the current directory instead of {0}".format(LOG_DIR), action="store_true")
   
   return parser.parse_args()
  
+# Appends the config directory to config file names
+def appendConfigDirectory(portable, file_name):
```

Comment:
Consider a more descriptive name, e.g. "CreateFullFilePath" or something like that.. Also "Append" indicates that the directory path is added at the end of what ever is being manipulated :)

Do this logic immediately in lines 53 and 55